From a6150cc7e064f009537e4a3a444a9eb83c667113 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 14 Jan 2015 15:26:58 +0000
Subject: [PATCH] abuild: fix move of .a file to -dev subpackage

seems like new version of busybox find would stop if lib/ dir does not
exist.
---
 abuild.in | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index e4e5091..68f6afd 100644
--- a/abuild.in
+++ b/abuild.in
@@ -1294,6 +1294,8 @@ default_dev() {
 	done
 
 	cd "$pkgdir" || return 0
+	local libdirs=usr/
+	[ -d lib/ ] && libdirs="lib/ $libdirs"
 	for i in usr/include usr/lib/pkgconfig usr/share/aclocal\
 			usr/share/gettext usr/bin/*-config	\
 			usr/share/vala/vapi usr/share/gir-[0-9]*\
@@ -1301,7 +1303,7 @@ default_dev() {
 			usr/lib/qt*/mkspecs			\
 			usr/lib/cmake				\
 			$(find . -name include -type d) 	\
-			$(find lib/ usr/ -name '*.[acho]' \
+			$(find $libdirs -name '*.[acho]' \
 				-o -name '*.prl' 2>/dev/null); do
 		if [ -e "$pkgdir/$i" ] || [ -L "$pkgdir/$i" ]; then
 			d="$subpkgdir/${i%/*}"	# dirname $i
-- 
2.2.2

